client/blocks/login/two-factor-authentication/waiting-notification-approval.jsx

```diff
@@ -40,8 +40,6 @@ class WaitingTwoFactorNotificationApproval extends Component {
 
 	componentWillReceiveProps( nextProps ) {
 		if ( ! this.props.pushSuccess && nextProps.pushSuccess ) {
-			const { translate } = this.props;
-			this.props.successNotice( translate( 'Logging In…' ) );
```

Comment:
That's probably a left-over of https://github.com/Automattic/wp-calypso/commit/06390be4396573e11125aec18026b3377eedbd11.